src/core/openthread-core-default-config.h

```diff
@@ -36,6 +36,46 @@
 #define OPENTHREAD_CORE_DEFAULT_CONFIG_H_
 
 /**
+ * @def OPENTHREAD_CONFIG_STACK_VENDOR_OUI
+ *
+ * The Organizationally Unique Identifier for the Thread stack.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_STACK_VENDOR_OUI
+#define OPENTHREAD_CONFIG_STACK_VENDOR_OUI                      0x18b430
```

Comment:
Perhaps this can be some obvious dummy value, such that folks bringing OT into production will noticed the need to set it to their OUI.